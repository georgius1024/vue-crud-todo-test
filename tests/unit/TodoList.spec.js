import { shallowMount } from "@vue/test-utils";
import component from "@/components/TodoList.vue";

describe("TodoList", () => {
  it("mounts without errors", () => {
    const wrapper = shallowMount(component, {
      props: {
        todos: [
          {
            _id: 101,
            name: "todo",
            done: true,
          },
        ],
      },
    });
    expect(wrapper.exists()).toBe(true);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
