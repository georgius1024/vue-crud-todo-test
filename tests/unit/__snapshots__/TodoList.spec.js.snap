// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoList mounts without errors 1`] = `
<div class="todo-list">
  <todo-item-stub todo="[object Object]"></todo-item-stub>
</div>
`;
